grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -262,7 +261,7 @@ public GrpcServiceBuilder addService(String path, BindableService bindableServic
      * The normal gRPC service path for the {@code Hello} method is
      * {@code "/example.grpc.hello.HelloService/Hello"}.
      * However if you set {@code "/foo"} to {@code path}, the {@code Hello} method will be served at
-     * {@code "/foo"}. This is useful for supporting unframed gRPC with HTTP idiomatic path.
+     * {@code "/foo/Hello"}. This is useful for supporting unframed gRPC with HTTP idiomatic path.
```

Comment:
The `path` should override the full method path (`/example.grpc.hello.HelloService/Hello`) with `/foo`.
I think the Javadoc is correct and the behavior should not be changed.